Apps/Database/src/Delegates/DBEmailDelegate.cs

```diff
@@ -72,7 +72,7 @@ public IList<Email> GetLowPriorityEmail(int maxRows)
             this.logger.LogTrace($"Getting list of low priority emails from DB... {maxRows}");
             IList<Email> retVal = this.dbContext.Email.Where(p => p.EmailStatusCode == EmailStatus.New &&
                                                    p.Priority < EmailPriority.Standard)
-                                        .OrderByDescending(s => s.Priority)
+                                        .OrderByDescending(o => o.Priority).ThenByDescending(o => o.CreatedDateTime)
```

Comment:
Shouldn't this be a `ThenByAscending(o => o.CreatedDateTime)` so that it processes from oldest to newest?